marklogic-data-hub/src/main/resources/ml-modules/transforms/hubEntitySearchTransform.sjs

```diff
@@ -0,0 +1,30 @@
+/**
+ Copyright (c) 2020 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+const entitySearchLib = require("/data-hub/5/entities/entity-search-lib.sjs");
+
+// Expects JSON content
+function transform(context, params, content) {
+  // TODO If entityName isn't defined, throw a nice error
```

Comment:
Need a nice error here - I think you can use the "throw"* functions in ds-util.sjs to get it done easily - e.g. throwBadRequest("entityName is required to generate search results containing entity properties");